speechmatics/client.py

```diff
@@ -435,7 +448,6 @@ async def run(
             consumer/producer tasks.
         """
         self.transcription_config = transcription_config
```

Comment:
Would this be a better place for checking if `transcription_config` is `None`, like the added asserts in lines 137 and 164? Did you find some circumstances where it was `None`?

Reply:
The assertions is mostly to satisfy any type checkers, to know that this is actually always a value.